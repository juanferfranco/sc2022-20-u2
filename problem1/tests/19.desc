crea varios eventos y los destruye del primero al Ãºltimo creado