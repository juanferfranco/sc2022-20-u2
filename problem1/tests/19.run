./main < tests/19.in
