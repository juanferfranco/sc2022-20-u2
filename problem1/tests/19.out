ev1
ev2
ev3
ev2
ev3
ev3
empty
